viewer/src/data/provider/BinaryFileProvider.ts

```diff
@@ -0,0 +1,8 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+export interface BinaryFileProvider {
+  fetchBinaryFileFromUrl(url: string): Promise<Uint8Array>;
+  fetchBinaryFileFromCdf(blobId: number, filePath: string): Promise<Uint8Array>;
```

Comment:
To me it still looks off that we have one provider that needs to know about all possible data sources, but I've been barking up this tree before.... 

Do we really need both this interface and `BinaryBlobFileProvider?

Reply:
No, BinaryBlobProvider is going away, I think I removed it in the latest push?
Yes, we are going back and forth on this issue. I don't think the BinaryFileProvider.ts is in use currently, so I could remove it for the current merge, last time we talked about this it seemed like you were for having a split at the repository level since it simplified the code structure, but I kinda have lost track xD

Reply:
![image](https://user-images.githubusercontent.com/6741854/81107575-b7ddaf00-8f17-11ea-8521-b2164326db4d.png)
Both classes are still present